### ISSUE TITLE
[Feedback]: Why only cover the screen sizing mode here and not also go over the Screen Size and Zoom settings?

### ISSUE THREAD

**Issue by pdxred** [Devolutions/doc#1147]
### Contact Details

TOBY@PDX.RED

### Feedback

[Enter feedback here]

 Why only cover the screen sizing mode here and not also go over the Screen Size and Zoom settings?

 ---
#### Document Details

⚠ *Do not edit this section. It is required for docs.devolutions.net ➟ GitHub issue linking.*

* Content: [Screen sizing mode in Remote Desktop Manager](https://docs.devolutions.net/kb/remote-desktop-manager/knowledge-base/screen-sizing-mode/)
* Content Source: [/content/en/kb/remote-desktop-manager/knowledge-base/screen-sizing-mode.md](https://github.com/Devolutions/doc/tree/master/docs/content/en/kb/remote-desktop-manager/knowledge-base/screen-sizing-mode.md)
* Service: **<b>Devolutions</b> Knowledge Base**

**Comment by JeanSebastienSimard** [comment]
Hello, 

Thank you for sharing your valuable feedback with us!

Your suggestion to include documentation on **_Remote desktop size_** and **_Zoom_** options is spot on. We're actually in the process of expanding our guides to cover a broader range of RDP and SSH settings. The focus on **_Screen sizing mode_** in the current KB article stems from a high volume of user requests, as many found it a bit complex and sought more clarity.

However, given your specific request for information on **_Remote desktop size_** and **_Zoom_**, I'm inclined to expedite their inclusion in our documentation. We will be exploring the possibility of integrating this new information into the existing KB article or crafting a new one to ensure comprehensive and user-friendly guidance.

Your input is instrumental in helping us enhance our user experience. We truly appreciate your involvement in making our resources more helpful for everyone.

Best regards,

Devolutions Documentation Team

**Comment by JeanSebastienSimard** [comment]
Hello again,

I just finished updating the screen sizing mode KB. It now encompasses all display properties in RDP sessions: screen sizing mode, remote desktop size, zoom, color settings, and connection bar settings. I also changed its title and URL, so here's the link to the new topic: [RDP entry display properties](https://docs.devolutions.net/kb/remote-desktop-manager/knowledge-base/rdp-entry-display-properties/).

Thank you again for bringing this to our attention. I hope these changes will make your use of Remote Desktop Manager more efficient and enjoyable.

Best regards, 

Devolutions Documentation Team
